### ISSUE TITLE
The transformed face looks too alien  How do I solve this?

### ISSUE THREAD

**Issue by facelabmania** [iperov/DeepFaceLab#897]
The transformed face looks too alien

How do I solve this?

  [0] : GeForce GTX 1650 SUPER

[0] Which GPU indexes to choose? :
0

[0] Autobackup every N hour ( 0..24 ?:help ) :
0
[n] Write preview history ( y/n ?:help ) : y
[n] Choose image for the preview history ( y/n ) : y
[0] Target iteration :
0
[y] Flip faces randomly ( y/n ?:help ) :
y
[8] Batch_size ( ?:help ) : 4
4
[128] Resolution ( 64-640 ?:help ) :
128
[f] Face type ( h/mf/f/wf/head ?:help ) :
f
[df] AE architecture ( ?:help ) :
df
[256] AutoEncoder dimensions ( 32-1024 ?:help ) :
256
[64] Encoder dimensions ( 16-256 ?:help ) :
64
[64] Decoder dimensions ( 16-256 ?:help ) :
64
[22] Decoder mask dimensions ( 16-256 ?:help ) :
22
[n] Eyes priority ( y/n ?:help ) :
n
[n] Uniform yaw distribution of samples ( y/n ?:help ) :
n
[y] Place models and optimizer on GPU ( y/n ?:help ) :
y
[n] Use learning rate dropout ( n/y/cpu ?:help ) :
n
[y] Enable random warp of samples ( y/n ?:help ) : n
[0.0] GAN power ( 0.0 .. 10.0 ?:help ) :
0.0
[0.0] 'True face' power. ( 0.0000 .. 1.0 ?:help ) :
0.0
[0.0] Face style power ( 0.0..100.0 ?:help ) :
0.0
[0.0] Background style power ( 0.0..100.0 ?:help ) :
0.0
[none] Color transfer for src faceset ( none/rct/lct/mkl/idt/sot ?:help ) :
none
[n] Enable gradient clipping ( y/n ?:help ) :
n
[n] Enable pretraining mode ( y/n ?:help ) :
n

**Comment by test1230-lab** [comment]
"The transformed face looks too alien" what do you mean

**Comment by facelabmania** [comment]
> "The transformed face looks too alien" what do you mean

https://github.com/iperov/DeepFaceLab/issues/898 chek it plz

**Comment by justinjohn0306** [comment]
> The transformed face looks too alien
> 
> How do I solve this?
> 
> [0] : GeForce GTX 1650 SUPER
> 
> [0] Which GPU indexes to choose? :
> 0
> 
> [0] Autobackup every N hour ( 0..24 ?:help ) :
> 0
> [n] Write preview history ( y/n ?:help ) : y
> [n] Choose image for the preview history ( y/n ) : y
> [0] Target iteration :
> 0
> [y] Flip faces randomly ( y/n ?:help ) :
> y
> [8] Batch_size ( ?:help ) : 4
> 4
> [128] Resolution ( 64-640 ?:help ) :
> 128
> [f] Face type ( h/mf/f/wf/head ?:help ) :
> f
> [df] AE architecture ( ?:help ) :
> df
> [256] AutoEncoder dimensions ( 32-1024 ?:help ) :
> 256
> [64] Encoder dimensions ( 16-256 ?:help ) :
> 64
> [64] Decoder dimensions ( 16-256 ?:help ) :
> 64
> [22] Decoder mask dimensions ( 16-256 ?:help ) :
> 22
> [n] Eyes priority ( y/n ?:help ) :
> n
> [n] Uniform yaw distribution of samples ( y/n ?:help ) :
> n
> [y] Place models and optimizer on GPU ( y/n ?:help ) :
> y
> [n] Use learning rate dropout ( n/y/cpu ?:help ) :
> n
> [y] Enable random warp of samples ( y/n ?:help ) : n
> [0.0] GAN power ( 0.0 .. 10.0 ?:help ) :
> 0.0
> [0.0] 'True face' power. ( 0.0000 .. 1.0 ?:help ) :
> 0.0
> [0.0] Face style power ( 0.0..100.0 ?:help ) :
> 0.0
> [0.0] Background style power ( 0.0..100.0 ?:help ) :
> 0.0
> [none] Color transfer for src faceset ( none/rct/lct/mkl/idt/sot ?:help ) :
> none
> [n] Enable gradient clipping ( y/n ?:help ) :
> n
> [n] Enable pretraining mode ( y/n ?:help ) :
> n

Show a preview

**Comment by facelabmania** [comment]
????

**Comment by test1230-lab** [comment]
> > The transformed face looks too alien
> 
> > 
> 
> > How do I solve this?
> 
> > 
> 
> > [0] : GeForce GTX 1650 SUPER
> 
> > 
> 
> > [0] Which GPU indexes to choose? :
> 
> > 0
> 
> > 
> 
> > [0] Autobackup every N hour ( 0..24 ?:help ) :
> 
> > 0
> 
> > [n] Write preview history ( y/n ?:help ) : y
> 
> > [n] Choose image for the preview history ( y/n ) : y
> 
> > [0] Target iteration :
> 
> > 0
> 
> > [y] Flip faces randomly ( y/n ?:help ) :
> 
> > y
> 
> > [8] Batch_size ( ?:help ) : 4
> 
> > 4
> 
> > [128] Resolution ( 64-640 ?:help ) :
> 
> > 128
> 
> > [f] Face type ( h/mf/f/wf/head ?:help ) :
> 
> > f
> 
> > [df] AE architecture ( ?:help ) :
> 
> > df
> 
> > [256] AutoEncoder dimensions ( 32-1024 ?:help ) :
> 
> > 256
> 
> > [64] Encoder dimensions ( 16-256 ?:help ) :
> 
> > 64
> 
> > [64] Decoder dimensions ( 16-256 ?:help ) :
> 
> > 64
> 
> > [22] Decoder mask dimensions ( 16-256 ?:help ) :
> 
> > 22
> 
> > [n] Eyes priority ( y/n ?:help ) :
> 
> > n
> 
> > [n] Uniform yaw distribution of samples ( y/n ?:help ) :
> 
> > n
> 
> > [y] Place models and optimizer on GPU ( y/n ?:help ) :
> 
> > y
> 
> > [n] Use learning rate dropout ( n/y/cpu ?:help ) :
> 
> > n
> 
> > [y] Enable random warp of samples ( y/n ?:help ) : n
> 
> > [0.0] GAN power ( 0.0 .. 10.0 ?:help ) :
> 
> > 0.0
> 
> > [0.0] 'True face' power. ( 0.0000 .. 1.0 ?:help ) :
> 
> > 0.0
> 
> > [0.0] Face style power ( 0.0..100.0 ?:help ) :
> 
> > 0.0
> 
> > [0.0] Background style power ( 0.0..100.0 ?:help ) :
> 
> > 0.0
> 
> > [none] Color transfer for src faceset ( none/rct/lct/mkl/idt/sot ?:help ) :
> 
> > none
> 
> > [n] Enable gradient clipping ( y/n ?:help ) :
> 
> > n
> 
> > [n] Enable pretraining mode ( y/n ?:help ) :
> 
> > n
> 
> 
> 
> Show a preview
> 
> 

It is in the issue he linked.

**Comment by ghost** [comment]
Issue solved / answered (or it seems like user error), please close it.
